[CI] MultiClusterSpecIT test {meta.CountFunctions} failing

**Build Scans:**
- [elasticsearch-intake #9471 / 8.16.0_bwc-snapshots](https://gradle-enterprise.elastic.co/s/awiyej7vum4do)
- [elasticsearch-intake #9470 / 8.16.0_bwc-snapshots](https://gradle-enterprise.elastic.co/s/3x747yszlpeuq)
- [elasticsearch-pull-request #30417 / 8.16.0_bwc-snapshots](https://gradle-enterprise.elastic.co/s/2koji6uwhmj7a)
- [elasticsearch-intake #9469 / 8.16.0_bwc-snapshots](https://gradle-enterprise.elastic.co/s/ejgqsudobkruq)
- [elasticsearch-pull-request #30184 / 8.16.0_bwc-snapshots](https://gradle-enterprise.elastic.co/s/utbwtg23sflai)
- [elasticsearch-pull-request #30033 / 8.16.0_bwc-snapshots](https://gradle-enterprise.elastic.co/s/kdhjr5wruq2d2)
- [elasticsearch-pull-request #30005 / 8.16.0_bwc-snapshots](https://gradle-enterprise.elastic.co/s/2mrmkz2mvv2yy)

**Reproduction Line:**
```
./gradlew ":x-pack:plugin:esql:qa:server:multi-clusters:v8.16.0#bwcTest" -Dtests.class="org.elasticsearch.xpack.esql.ccq.MultiClusterSpecIT" -Dtests.method="test {meta.CountFunctions}" -Dtests.seed=DB5C696673FCF85F -Dtests.bwc=true -Dtests.locale=sv-FI -Dtests.timezone=America/Indiana/Marengo -Druntime.java=22
```

**Applicable branches:**
main

**Reproduces locally?:**
N/A

**Failure History:**
[See dashboard](https://es-delivery-stats.elastic.dev/app/dashboards#/view/dcec9e60-72ac-11ee-8f39-55975ded9e63?_g=(refreshInterval:(pause:!t,value:60000),time:(from:now-7d%2Fd,to:now))&_a=(controlGroupInput:(chainingSystem:HIERARCHICAL,controlStyle:twoLine,ignoreParentSettings:(ignoreFilters:!f,ignoreQuery:!f,ignoreTimerange:!f,ignoreValidations:!t),panels:('0c0c9cb8-ccd2-45c6-9b13-96bac4abc542':(explicitInput:(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,enhancements:(),fieldName:task.keyword,grow:!t,id:'0c0c9cb8-ccd2-45c6-9b13-96bac4abc542',searchTechnique:wildcard,selectedOptions:!(),singleSelect:!t,title:'Gradle%20Task',width:medium),grow:!t,order:0,type:optionsListControl,width:small),'144933da-5c1b-4257-a969-7f43455a7901':(explicitInput:(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,enhancements:(),fieldName:name.keyword,grow:!t,id:'144933da-5c1b-4257-a969-7f43455a7901',searchTechnique:wildcard,selectedOptions:!('test%20%7Bmeta.CountFunctions%7D'),title:Test,width:medium),grow:!t,order:2,type:optionsListControl,width:medium),'4e6ad9d6-6fdc-4fcc-bf1a-aa6ca79e0850':(explicitInput:(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,enhancements:(),fieldName:className.keyword,grow:!t,id:'4e6ad9d6-6fdc-4fcc-bf1a-aa6ca79e0850',searchTechnique:wildcard,selectedOptions:!('org.elasticsearch.xpack.esql.ccq.MultiClusterSpecIT'),title:Suite,width:medium),grow:!t,order:1,type:optionsListControl,width:medium)))))

**Failure Message:**
```
java.lang.AssertionError: Data mismatch:
row 0 column 0:a list containing
row 0 column 0:0: expected "116" but was "117"
row 0 column 1:a list containing
row 0 column 1:0: expected "116" but was "117"
row 0 column 2:a list containing
row 0 column 2:0: expected "116" but was "117"
```

**Issue Reasons:**
- [main] 3 consecutive failures in pipeline elasticsearch-intake
- [main] 7 failures in test test {meta.CountFunctions} (1.8% fail rate in 381 executions)
- [main] 7 failures in step 8.16.0_bwc-snapshots (2.7% fail rate in 258 executions)
- [main] 3 failures in pipeline elasticsearch-intake (4.3% fail rate in 70 executions)
- [main] 4 failures in pipeline elasticsearch-pull-request (2.1% fail rate in 194 executions)

**Note:**
This issue was created using new test triage automation. Please report issues or feedback to es-delivery.

--- a/muted-tests.yml
+++ b/muted-tests.yml
@@ -195,15 +195,6 @@ tests:
   issue: https://github.com/elastic/elasticsearch/issues/112382
 - class: org.elasticsearch.xpack.eql.EqlClientYamlIT
   issue: https://github.com/elastic/elasticsearch/issues/112617
-- class: org.elasticsearch.xpack.esql.ccq.MultiClusterSpecIT
-  method: test {meta.CountFunctions}
-  issue: https://github.com/elastic/elasticsearch/issues/112659
-- class: org.elasticsearch.xpack.esql.qa.mixed.MixedClusterEsqlSpecIT
-  method: test {meta.CountFunctions SYNC}
-  issue: https://github.com/elastic/elasticsearch/issues/112660
-- class: org.elasticsearch.xpack.esql.qa.mixed.MixedClusterEsqlSpecIT
-  method: test {meta.CountFunctions ASYNC}
-  issue: https://github.com/elastic/elasticsearch/issues/112661
 
 # Examples:
 #

--- a/x-pack/plugin/esql/qa/testFixtures/src/main/resources/meta.csv-spec
+++ b/x-pack/plugin/esql/qa/testFixtures/src/main/resources/meta.csv-spec
@@ -516,5 +516,5 @@ countFunctions#[skip:-8.15.99]
 meta functions |  stats  a = count(*), b = count(*), c = count(*) |  mv_expand c;
 
 a:long | b:long | c:long
-116    | 116    | 116
+117    | 117    | 117
 ;

